packages/client/src/products/types/facetGroup.types.ts

```diff
@@ -57,4 +57,5 @@ export type FacetGroup = {
   _isClearHidden: boolean;
   _isClosed: boolean;
   dynamic: number;
+  hash: string;
```

Comment:
The hash will only be added to the FacetGroup entity in redux. This type is defined in the client package which can be used without redux. You would need to change the FacetGroup type inside the listing to account for that but I think we should not expose the hash to the consumers as it is an implementation detail.

Reply:
Agree 👍 updated!